docs/rsync.md

```diff
@@ -0,0 +1,23 @@
+# rsync
+
+[Source](/src/Utility/Rsync.php)
+
+## IMPORTANT
+
+This is the deployer's built-in rsync and must not be confused with `/contrib/rsync.php`. Their configuration options are also very different, read carefully below.
+
+## Configuration options
```

Comment:
Better to put it next to upload/download func.

Reply:
ok, what is the syntax to link from `api.md` to `rsync.md`?

Reply:
@antonmedv any idea? thanks

Reply:
[TEXT](LINK TARGET)
e.g.[Source](/src/Utility/Rsync.php)

Does this help?

Reply:
Also I don’t like wording. It’s not an external docs. “Their” not fit here.

Reply:
@Schrank I'm not sure, I could not find any references around, but let me try
@antonmedv Let me find some time to fix it all up